src/components/ReceiveLandingPageComponent.jsx

```diff
@@ -49,7 +49,7 @@ class ReceiveLandingPageComponent extends Component {
   }
 
   render () {
-    const { actionsPending, transfer, classes, sendTime, receiveTime, cancelTime } = this.props
+    const { actionsPending, transfer, classes, sendTime, receiveTime, cancelTime, toCurrencyAmount } = this.props
```

Comment:
Move toCurrencyAmount function to container, pass down the result to component. 
This applies to the rest of the PR